tests/console/salt.pm

```diff
@@ -33,7 +33,7 @@ EOF
     sleep(30);
     assert_script_run("salt-key --accept-all -y");
     record_soft_failure 'bsc#1069711';    # Added 120s to ping minion
-    validate_script_output "salt '*' test.ping -t 120 | grep -woh True > /dev/$serialdev", sub { m/True/ }, 120;
+    validate_script_output "salt '*' test.ping -t 180 | grep -woh True > /dev/$serialdev", sub { m/True/ }, 180;
```

Comment:
As we increase this timeout for the third time as far as I remember, I guess we should at least add soft-failure here. And again, there is no guarantee that 180 seconds will be enough.

Reply:
soft-failure is added in the line before, there are currently two soft failures for this job referencing the same bug. Yes I agree, there is no guarantee. Not sure if there is a better way to add the soft failure.